res/lang/programme/programme_en.txt

```diff
@@ -106,46 +106,46 @@ PROGRAMME_PRODUCT_SHOW        = Show
 PROGRAMME_PRODUCT_EVENT       = Event
-PROGRAMME_PRODUCT_MISC        = Misc
+PROGRAMME_PRODUCT_MISC        = Misc.
 
 PROGRAMME_IN_PROGRAMME_PLAN              = Programme in programme plan!
 TRAILER_IN_PROGRAMME_PLAN                = Trailer in programme plan!
 PROGRAMME_AND_TRAILER_IN_PROGRAMME_PLAN  = Programme and trailer in programme plan!
 ONLY_1_BROADCAST_POSSIBLE                = Only 1 broadcast possible!
 ONLY_X_BROADCASTS_POSSIBLE               = Only %X% broadcasts possible!
 NO_MORE_BROADCASTS_ALLOWED               = No more broadcasts allowed!
-BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted
+BROADCAST_TIME_RESTRICTED                = Time of broadcast restricted!
 BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y       = Broadcast only allowed from %X%:00 to %Y%:00!
 FIRST_BROADCAST_ONLY_ALLOWED_FROM_X_TO_Y = First broadcast only allowed from %X%:00 to %Y%:00!
 LIVE_PRODUCTION_FINISH_WILL_COST_X       = Live production finish will cost %X%
 
 OPENINGSHOW_TITLE       = Big launch of "%CHANNELNAME%"
-OPENINGSHOW_DESCRIPTION = Let's pop the corks. %CHANNELNAME% is launching today.
+OPENINGSHOW_DESCRIPTION = Let's pop the corks! %CHANNELNAME% is launching today.
 
 SPORT_PROGRAMME_TITLE                                = %SPORTNAME%: %LEAGUENAME% %SEASONYEARSTART%
 SPORT_PROGRAMME_ALL_X_MATCHES_OF_LEAGUEX_IN_SEASON_X = All %MATCHCOUNT% matches of %LEAGUENAME% in season %SEASONYEARSTART%.
 SPORT_PROGRAMME_MATCH_TIMES                          = Match times
-SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between place %TEAM1RANK% and %TEAM2RANK% of the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
-SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star %TEAM1STARSHORT% against the team around top scorer %TEAM2STAR%.
+SPORT_PROGRAMME_MATCH_DESCRIPTION1                   = Match between teams placed %TEAM1RANK% and %TEAM2RANK% in the league. %TEAM1NAMEINITIALS% coach %TEAM1TRAINERSHORT% is optimistic.
+SPORT_PROGRAMME_MATCH_DESCRIPTION2                   = %TEAM1NAMEINITIALS% star, %TEAM1STARSHORT%, against top scorer %TEAM2STAR%'s team.
 SPORT_PROGRAMME_MATCH_DESCRIPTION3                   = An interesting game is coming up when %TEAM1ARTICLE1% %TEAM1NAME% meet the players of %TEAM2NAMEINITIALS%.
 SPORT_PROGRAMME_MATCH_DESCRIPTION4                   = %TEAM2ARTICLE1% %TEAM2NAME% visiting %TEAM1NAMEINITIALS% in %TEAM1CITY%.
-SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, rank %TEAM2RANK% in the league.
+SPORT_PROGRAMME_MATCH_DESCRIPTION5                   = %TEAM2TRAINER% and his team travel to %TEAM2CITY%, placed %TEAM2RANK% in the league.
 
-SPORT_PROGRAMME_PLAYOFF_MATCH    = Playoff match.
-SPORT_PROGRAMME_MATCH_OF_LEAGUEX = Match of %LEAGUENAME%.
-SPORT_PLAYOFFS_LONG              = "Playoffs"
-SPORT_PLAYOFFS_SHORT             = "PO"
+SPORT_PROGRAMME_PLAYOFF_MATCH    = Relegation match.
```

Comment:
In this case this could work, as the German "Bunte Liga" (you know..) has "relegations". Other Soccer leagues might have "playoffs". And As we do not just have "Soccer" but also "ice hockey" (using the same locales here) we have "Playoffs" there and no relegation games.

Reply:
I don't really get this, the German and French here refer to "relegations" but what are "relegations", are they teams playing at the end of the season to avoid relegation?  If that's the case "playoff" sounds wrong in English as that's always a competition to get a promotion.

Reply:
Yeah but I mean you may get "relegation playoffs" but in general, playoffs are for promotion so if you just say "playoffs" without other context it sounds like promotion playoffs.  Are these news pieces supposed to be referring to promotion playoffs or relegation playoffs?

Reply:
For me playoffs is the way to "go up" and the relegation is the "chance to avoid going downwards".
I am not much into soccer, so am free to whatever others suggest.

But keep in mind: we also talk about ice hockey, basketball and other games - so either split across sport kinds or we need to find a "suiting for all" term.

Reply:
Yeah but what I'm asking is, what does the game intend here?  Is it actually talking about relegation playoffs in its news articles, or promotion playoffs?  Or a mixture of both?

Reply:
As said - I am not much into soccer. For me the "playoffs" are all the matches happening between last slots of the "upper" league versus first ranks of "lower" league.
Loosers move down a league, Winners move up.

Reply:
Well if "playoff" is appropriate then it seems like programme_de and programme_fr are wrong then:
"SPORT_PROGRAMME_PLAYOFF_MATCH    = Relegationsspiel.
SPORT_PROGRAMME_PLAYOFF_MATCH    = Match de relégation.

Reply:
If you are more profound of soccer terms than me (I guess most people are :D) then feel free to tell (read: if you are unsure as I am, then maybe others can chime in :)). If you think "Relegationsspiel" fits best, and the "relegation matches" are the same - then we of course take your new suggested localization.

Reply:
In UK football leagues, playoffs happen amongst the top teams of a lower league to decide who gets promoted; they don't play against lower teams in the higher league.  So it seems like the English and German leagues work differently.  I guess "playoffs" may be the best translation in light of this.

Reply:
I think "basketball" has a similar (but not the same) approach than the German soccer leagues ... best of "lower leagues" against "worst of upper leagues" (in Basketball they fight first for the promotion you described).

Hmm, hope there are no too "strict" soccer enthusiasts around arguing about the "odd system" these German dudes "simulate" in their game...